src/content/blog/2025-10-03-whats-coming-in-eslint-10.0.0.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title: What's coming in ESLint v10.0.0
+teaser: "We want to share with you the significant changes that will be coming in v10.0.0 so you can better prepare for the changes and understand why the changes are necessary."
+tags:
+  - major
+  - breaking changes
+authors:
+  - nzakas
+categories:
+  - API Changes
+---
+
+The Technical Steering Committee (TSC) has finalized the features for ESLint v10.0.0. This post outlines our high-level plans for v10.0.0. For more details, and to keep up to date with everything that is planned for v10.0.0, visit our [project board](https://github.com/orgs/eslint/projects/6).
+
+## Development plan
+
+Similar to v9.0.0, we will develop v10.0.0 in phases to ensure stability and gather community feedback:
+
+1. **Alpha.** The alpha release will include the most significant breaking changes that we expect will cause the most disruption for existing users. This early release will help us gather feedback and validate our approach.
+2. **Beta.** The beta release will include the remaining features and smaller breaking changes that will impact fewer users.
+
+After the beta has been validated through community testing, we will publish one or more release candidates as we continue to fix bugs and address compatibility issues.
+
+## Significant changes in v10.0.0-alpha
+
+The following changes are planned for the alpha release and represent significant breaking changes.
+
+### Dropping support for Node.js < v20.19.0
+
+Following our Node.js version support policy for major releases, ESLint v10.0.0 will [require Node.js v20.19.0 or later](https://github.com/eslint/eslint/issues/19969). The new supported versions will be:
+
+* Node.js v20.19.0 and above (Maintenance LTS)
+* Node.js v22.13.0 and above (Active LTS)
+* Node.js v24 and above (Current)
```

Comment:
Do you want to mention Node.js v25 here which is expected to release on Oct 15, 2025?

Or maybe say something generic like ESlint 10 will track other Node.js transitions / releases as they occur (if that is what you intend). For instance Node.js v24 is planned to transition to Active LTS on Oct 28, 205.